.eslintrc.js

```diff
@@ -1,10 +1,11 @@
 module.exports = {
-  extends: '@mate-academy/eslint-config',
+  extends: "@mate-academy/eslint-config",
   env: {
-    jest: true
+    jest: true,
   },
   rules: {
-    'no-proto': 0
+    "no-proto": 0,
+    quotes: "off",
```

Comment:
The `quotes` rule is set to "off", which means there is no enforcement on the usage of single or double quotes. Consider setting this rule to enforce consistent quote usage across your codebase.